src/main/java/hsleiden/stenentijdperk/stenentijdperk/Controllers/BoardController.java

```diff
@@ -363,7 +363,6 @@ private boolean resourcesBetalen(List<Integer> kost) {
         }
     }
 
-    // TODO tijdelijk
```

Comment:
Als het tijdelijk is, en er staat een TODO bij, waarom haal je dan niet ook de functie weg. Als de functie nog gebruikt wordt, haal dan ook niet de TODO weg.